src/open_company/api/companies.clj

```diff
@@ -96,26 +100,75 @@
   :delete! (fn [_] (company/delete-company slug))
 
   ;; Create or update a company
+  ;; TODO remove possibility to create company
```

Comment:
What's the benefit of providing two things to achieve the same thing? Functionality seems identical as far as I can tell?